src/nnsight/models/LanguageModel.py

```diff
@@ -175,6 +175,9 @@ def _load(
             if tokenizer_kwargs is None:
                 tokenizer_kwargs = {}
 
+            if "padding_side" not in tokenizer_kwargs:
+                tokenizer_kwargs["padding_side"] = "left"
+
```

Comment:
maybe add a warning if the set kwarg is not left explaining why it could go wrong?